src/UserService.Mappers/Models/CertificateInfoMapper.cs

```diff
@@ -2,30 +2,30 @@
 using LT.DigitalOffice.UserService.Models.Db;
 using LT.DigitalOffice.UserService.Models.Dto.Enums;
 using LT.DigitalOffice.UserService.Models.Dto.Models;
-using System;
+using System.Collections.Generic;
 
 namespace LT.DigitalOffice.UserService.Mappers.Models
 {
-    public class CertificateInfoMapper : ICertificateInfoMapper
+  public class CertificateInfoMapper : ICertificateInfoMapper
+  {
+    public CertificateInfo Map(
+      DbUserCertificate dbUserCertificate,
+      List<ImageInfo> images)
     {
-        public CertificateInfo Map(
-            DbUserCertificate dbUserCertificate,
-            ImageInfo image)
-        {
-            if (dbUserCertificate == null)
-            {
-                throw new ArgumentNullException(nameof(dbUserCertificate));
-            }
+      if (dbUserCertificate == null)
+      {
+        return null;
+      }
 
-            return new CertificateInfo
-            {
-                Id = dbUserCertificate.Id,
-                Name = dbUserCertificate.Name,
-                EducationType = (EducationType)dbUserCertificate.EducationType,
-                ReceivedAt = dbUserCertificate.ReceivedAt,
-                SchoolName = dbUserCertificate.SchoolName,
-                Image = image
-            };
-        }
+      return new CertificateInfo
+      {
+        Id = dbUserCertificate.Id,
+        Name = dbUserCertificate.Name,
+        EducationType = (EducationType)dbUserCertificate.EducationType,
```

Comment:
do not convert here, you should use [JsonConverter(typeof(StringEnumConverter))]